### PR TITLE
Create Storebot.me.xml

### PR DESCRIPTION
**Pull request by davidabian**

### CODE REVIEW

File: src/chrome/content/rules/Storebot.me.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!--
+	Ruleset for storebot.me.
+
+	Not supporting HTTPS or using an invalid certificate:
+		* dev.storebot.me
+		* email.storebot.me
+		* mail.storebot.me
```
Comment (J0WI) [inline src/chrome/content/rules/Storebot.me.xml]: idea.storebot.me

File: src/chrome/content/rules/Storebot.me.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!--
+	Ruleset for storebot.me.
+
+	Not supporting HTTPS or using an invalid certificate:
```
Comment (J0WI) [inline src/chrome/content/rules/Storebot.me.xml]: Could you be more verbose about the error (e.g. mismacth, slef-signed, https refused...)?

**Comment by jeremyn** [comment]
@davidabian Are you still interested in working on this one?

**Comment by davidabian** [comment]
Yes, I am.

**Comment by jeremyn** [comment]
Great!

- [ ] Please change the [top comment](https://github.com/davidabian/https-everywhere/blob/756096ab2737975494793ced27215c7e73ced28a/src/chrome/content/rules/Storebot.me.xml#L1-L11) to this:
```xml
<!--
	Invalid certificate:
		dev.storebot.me
		email.storebot.me
		idea.storebot.me

	Secure connection failed:
		mail.storebot.me

-->
```
This makes several changes, particularly removing unnecessary text and changing `Refused` to `Secure connection failed`. `Refused` is when the browser reports something like "Unable to connect", while `Secure connection failed` is more like what I see with https://mail.storebot.me .

**Comment by jeremyn** [comment]
Thanks, merged.

**Pull request merged**